src/hoc/withEditing/withEditing.tsx

```diff
@@ -31,19 +35,23 @@ export const withEditing = (config: IWithEditing) => (Component: React.Component
     });
 
     @ReduxConnect(mapStateToProps, mapDispatchToProps)
-    class ComponentWithEditing extends React.Component<any, any> {
+    class ComponentWithEditing extends React.Component<IWithEditingProps> {
+        componentDidMount() {
+            callIfDefined(this.props.toggleDirtyComponent, config.isDirty);
+        }
+
         componentWillUnmount() {
             callIfDefined(this.props.toggleDirtyComponent, false);
         }
 
         render() {
             return (
-                <div className={styles.stickyFooterPadding}>
-                    <Component {..._.omit(this.props, 'isDirty', 'footerChildren')}>
+                <div>

```

Comment:
we can use fragment ? I dont know if react-vapor support this new feature

Reply:
React-Vapor supports it, but the admin doesn't :crying_cat_face: